examples/matmul/matmul.cc

```diff
@@ -87,9 +87,10 @@ int main() {
 	multiply(queue, mat_a_buf, mat_b_buf, mat_c_buf);
 	multiply(queue, mat_b_buf, mat_c_buf, mat_a_buf);
 
-	bool verification_passed = true;
-	verify(queue, mat_a_buf, verification_passed);
-	queue.slow_full_sync(); // Wait for verification_passed to become available
+	celerity::buffer<bool> passed_buf(1);
```

Comment:
It might make sense to init this buffer to `false`, and also remove `no_init` in the accessor.  
That way the verification would fail for sure if for whatever reason the kernel were to not run at all.

Reply:
Done.